internal/supervisor/installation.go

```diff
@@ -1618,6 +1618,15 @@ func (s *InstallationSupervisor) manageInstallationResourcesCache() {
 			cacheLogger.WithError(err).Warn("Failed to query clusters")
 		} else {
 			for _, cluster := range clusters {
+				// Filter out a few states where we shouldn't try to obtain
```

Comment:
What about other state such as `provisioning-failed` or `upgrade-failed` etc? Perhaps we should only calculate the resources when the cluster is `stable`?

Reply:
Hmm, in theory the kubernetes API should be online and functioning even during these states. I know that installations won't be scheduled on clusters in these states, but we should be able to retrieve the resource totals still. I figured it was good to continue to monitor them so that if a dramatic change in resources available occurred during the non-stable time window that we captured that in the cache. What do you think?

Reply:
Yep, this makes sense. Thanks for the idea. I will add it to be safe.